### DISCUSSION TITLE
The first photo ever uploaded to the internet was of what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#670] [Galaxy Trivia] [answered]
**Bonus:** When did it happen?

**Comment by peterbaccaro** [comment, marked as answer]
The first picture uploaded on the web was the photo that captured **four slickly dressed women**.  It was posted by Tim Burners Lee, the inventor of the World Wide Web. The four women belonged to an all-female comedy band called Les Horrible Cernettes – or the Horrible CERN Girls. It was uploaded on **18 July, 1992**.

**Comment by SANKARAMDAS** [comment]
A Women's photo.
